renovate.json

```diff
@@ -2,8 +2,6 @@
 	"extends": [
 	"config:base"
 	],
-	"baseBranches": ["main", "3.x"],
-	"useBaseBranchConfig": "merge",
```

Comment:
I did it because forking-renovate does not support these configs as is. Maybe it'll work if we run it separately on both the branches. That's why I have also updated the renovate.json in 3.x with full config, earlier it was just an addition on top of the main branch config. This workflow should ideally run on both the branches totally separately.